ui/apps/platform/src/Containers/Vulnerabilities/WorkloadCves/Overview/WorkloadCvesOverviewPage.tsx

```diff
@@ -152,6 +153,15 @@ const searchFilterConfigWithFeatureFlagDependency = [
     clusterSearchFilterConfig,
 ];
 
+const defaultStorage: VulnMgmtLocalStorage = {
```

Comment:
This value was once dependent on a feature flag. Now that it is not, we can pull it out of the component's body.